Helpjuice/ckeditor5
# [Editor] [sc-1316] Drag-and-drop image on CKEditor should upload the file and embed it in the article

## Describe the changes in this pull request

This PR removes the SimpleUpload configuration and adds a missing image type in the allowed types to be dropped in the editor.

### How should this be tested during the review?

Check the app's PR for more details: https://github.com/hajrice/helpjuice4/pull/2498
